examples/linked_consumer/test_pkg_deps_linked.js

```diff
@@ -21,7 +21,17 @@ try {
     e = ex
 }
 if (e == null) {
-    throw new Error('devDependency should NOT be available')
+    // TODO: see https://github.com/aspect-build/rules_js/issues/2013 and similar issues.
```

Comment:
<v9 the lockfile `dev` flag was used (incorrectly) for this feature, >=v9 that flag no longer exists so the (buggy) behaviour is gone but we have nothing replacing it yet...